tox.ini

```diff
@@ -0,0 +1,53 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+skipsdist = true
+envlist =
+    py35-sqlite
+    flake8
+    docs
+    isort
+
```

Comment:
@sergei-maertens I am going to leave this out for now. Currently the only time you need Python 2 installed is for the docs on Mac OS X, and that requirement will go away as soon as pyenchant 1.6.7 is released. It's required that folks run the tests on at least one version of Python 3. Adding `skip_missing_interpreters = true` introduces the odd case where I can explicitly specify a Python version, and the build still "succeeds" because the interpreter doesn't exist:

	$ tox -e py34-sqlite
	py34-sqlite create: /Users/tobias/caktus/django/.tox/py34-sqlite
	ERROR: InterpreterNotFound: python3.4
	______________________________________________________ summary ______________________________________________________
	SKIPPED:  py34-sqlite: InterpreterNotFound: python3.4
	  congratulations :)
    $ echo $?
    0